src/index.ts

```diff
@@ -162,8 +188,24 @@ export function createIframe({
 
   const iframeSrc = new URL(targetPages[targetPage])
   iframeSrc.searchParams.append('embedding-org-id', normalisedOrgId)
+  // This device-id method is deprecated, it should be removed in a future version.
   if (deviceId) iframeSrc.searchParams.append('device-id', deviceId)
   if (disableChat) iframeSrc.searchParams.append('disable-chat', 'true')
+  if (disableHelp) iframeSrc.searchParams.append('disable-help', 'true')
+  if (themeOptions) {
+    const themeOptionsEncoded = encodeURIComponent(btoa(JSON.stringify(themeOptions)))
+    iframeSrc.searchParams.append('theme-options', themeOptionsEncoded)
+  }
+  if (analyticsInfo) {
+    const analyticsInfoEncoded = encodeURIComponent(btoa(JSON.stringify(analyticsInfo)))
+    iframeSrc.searchParams.append('analytics-info', analyticsInfoEncoded)
+
+    if (analyticsInfo.deviceId) {
+      // This will override the previous device-id above, but that's what we want.
```

Comment:
Can you confirm the override works?

I don't expect it to work because you're using `append`, and lobby only looks at the first query value [here](https://github.com/getsetup-io/lobby/blob/898e469c1e52dbe6696724911ff0f87f9621a22d/pages/_app.tsx#L51).

Reply:
Yeah, that should have been `.set()`. All of these `.append()`s should probably be swapped for sets, but we'll do that later.